Failed tests (10905): TestStoreRangeDownReplicate: infinite NotLeaderError on ChangeReplica

The following test appears to have failed:

[#10905](https://circleci.com/gh/cockroachdb/cockroach/10905):

```
W0109 12:21:53.446111 921 kv/dist_sender.go:694  range 1: replica node_id:3 store_id:3 replica_id:3  not leader; leader is node_id:1 store_id:1 replica_id:1 
W0109 12:21:53.453696 921 kv/dist_sender.go:636  failed to invoke BeginTransaction [/Local/Range/RangeDescriptor/"m",/Min), ConditionalPut [/Local/Range/RangeDescriptor/"m",/Min), Put [/Meta2/Max,/Min): range 1: replica node_id:3 store_id:3 replica_id:3  not leader; leader is node_id:1 store_id:1 replica_id:1 
W0109 12:21:53.454063 921 kv/dist_sender.go:694  range 1: replica node_id:3 store_id:3 replica_id:3  not leader; leader is node_id:1 store_id:1 replica_id:1 
W0109 12:21:53.460816 921 kv/dist_sender.go:636  failed to invoke BeginTransaction [/Local/Range/RangeDescriptor/"m",/Min), ConditionalPut [/Local/Range/RangeDescriptor/"m",/Min), Put [/Meta2/Max,/Min): range 1: replica node_id:3 store_id:3 replica_id:3  not leader; leader is node_id:1 store_id:1 replica_id:1 
W0109 12:21:53.461146 921 kv/dist_sender.go:694  range 1: replica node_id:3 store_id:3 replica_id:3  not leader; leader is node_id:1 store_id:1 replica_id:1 
--- FAIL: TestStoreRangeDownReplicate (31.45s)
W0109 12:21:53.463938 921 kv/dist_sender.go:636  failed to invoke BeginTransaction [/Local/Range/RangeDescriptor/"m",/Min), ConditionalPut [/Local/Range/RangeDescriptor/"m",/Min), Put [/Meta2/Max,/Min): range 1: replica node_id:3 store_id:3 replica_id:3  not leader; leader is node_id:1 store_id:1 replica_id:1 
panic: timed out during shutdown [recovered]
	panic: timed out during shutdown [recovered]
	panic: timed out during shutdown

goroutine 4512 [running]:
testing.tRunner.func1(0xc820158360)
	/tmp/workdir/go/src/testing/testing.go:450 +0x1f5
github.com/cockroachdb/cockroach/util/leaktest.AfterTest(0x7fd40293c120, 0xc820158360)
	/go/src/github.com/cockroachdb/cockroach/util/leaktest/leaktest.go:114 +0x1a7
github.com/cockroachdb/cockroach/storage_test.(*multiTestContext).Stop(0xc82000c780)
	/go/src/github.com/cockroachdb/cockroach/storage/client_test.go:283 +0x2f9
github.com/cockroachdb/cockroach/storage_test.TestStoreRangeDownReplicate(0xc820158360)
	/go/src/github.com/cockroachdb/cockroach/storage/client_raft_test.go:719 +0xc1b
testing.tRunner(0xc820158360, 0x21390c8)
	/tmp/workdir/go/src/testing/testing.go:456 +0xdd
created by testing.RunTests
	/tmp/workdir/go/src/testing/testing.go:561 +0xaa4

goroutine 1 [chan receive]:
testing.RunTests(0x1a844c8, 0x2137fa0, 0xdb, 0xdb, 0x1)
	/tmp/workdir/go/src/testing/testing.go:562 +0xafa
testing.(*M).Run(0xc8204abf10, 0x1)
	/tmp/workdir/go/src/testing/testing.go:494 +0xe5
github.com/cockroachdb/cockroach/util/leaktest.TestMainWithLeakCheck(0xc8204abf10)
	/go/src/github.com/cockroachdb/cockroach/util/leaktest/leaktest.go:37 +0x2f
github.com/cockroachdb/cockroach/storage_test.TestMain(0xc8204abf10)
	/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:36 +0x34
main.main()
	github.com/cockroachdb/cockroach/storage/_test/_testmain.go:498 +0x20a

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/tmp/workdir/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 5 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x21931e0)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1029 +0x76
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:610 +0x117

goroutine 4563 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc820291560, 0xece22f242, 0x1030310, 0x2192c60, 0xc8204a6060, 0xc820052850, 0xc8200a4460, 0xc82016c1c0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x641
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc8200a4460, 0xc8204a65d0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:217 +0xb0
github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc8204a6660, 0xc8204a7d70)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:225 +0x45c
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:219 +0x452
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc820498a60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4582 [chan receive]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8202a68c0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 4614 [chan receive]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8202a6f50)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 4549 [chan receive]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc820498740)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4557 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc8204989a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4558 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc8204989c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4545 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:556 +0x15d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc820522bc0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4627 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc82044d800, 0x7fd40294c4a8, 0xc8203e8780, 0xc820358b40)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:300 +0xb5
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc82044d800, 0x7fd40294c4a8, 0xc8203e8780, 0xc820358b40)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:207 +0x4d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:208 +0xa6c

goroutine 4541 [chan receive]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8202a65a0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 4852 [IO wait]:
net.runtime_pollWait(0x7fd402941ae0, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8203c9250, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8203c9250, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc8203c91f0, 0xc82039d000, 0x1000, 0x1000, 0x0, 0x7fd402936050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc8200269d8, 0xc82039d000, 0x1000, 0x1000, 0x7fd4008a7b60, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
net/rpc.(*Client).input(0xc8203cec60)
	/usr/local/go/src/net/rpc/client.go:109 +0x177
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0x126

goroutine 4575 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820498e00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4570 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc8202fecc0, 0xece22f242, 0x209f7cf, 0x2192c60, 0xc8204a6060, 0xc8202fb110, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x641
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:213 +0x1b0
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc820498ca0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4585 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1559 +0xb0c
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc820535b30)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4576 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820498e60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4598 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc8205232c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4550 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1559 +0xb0c
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc82049b720)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4607 [chan receive]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc820523880)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4553 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:632 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc8204987e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4574 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820498de0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4542 [IO wait]:
net.runtime_pollWait(0x7fd402941720, 0x72, 0x7fd402936050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8202faae0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8202faae0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc8202faa80, 0x0, 0x7fd402941e08, 0xc820498f80)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc8200260b8, 0xc8203f7d20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc820560bd0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4594 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc8205231c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4567 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc820498c40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4616 [chan receive]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc820523ce0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4603 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc82044d4a0, 0xece22f242, 0x1a6f6e7, 0x2192c60, 0xc8204a6060, 0xc8202fa930, 0x0, 0x600000000)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x641
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:213 +0x1b0
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc8205233a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4571 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820498d40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4551 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:556 +0x15d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc8204987a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4619 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:613 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820523d40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4584 [chan receive]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc820523080)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4573 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820498dc0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4592 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc820523180)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4611 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:632 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc820523940)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4577 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc8202ff620, 0xece22f242, 0x238ea68, 0x2192c60, 0xc8204a6060, 0xc8202fb490, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x641
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:213 +0x1b0
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820498e80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4566 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc820498c20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4649 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc8202b73e0, 0x7fd40294c4a8, 0xc8201f3c80, 0xc820101620)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:300 +0xb5
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc8202b73e0, 0x7fd40294c4a8, 0xc8201f3c80, 0xc820101620)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:207 +0x4d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:208 +0xa6c

goroutine 4559 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc8204989e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4597 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc8205232a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4568 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc820498c60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4569 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc820498c80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4615 [IO wait]:
net.runtime_pollWait(0x7fd4029418a0, 0x72, 0x7fd402936050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8202fb640, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8202fb640, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc8202fb5e0, 0x0, 0x7fd402941e08, 0xc820576a00)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc820026120, 0xc8204b0d20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc8205509c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4543 [chan receive]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc820522b80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4588 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:632 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc820523120)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4617 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1559 +0xb0c
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820512b30)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4618 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:556 +0x15d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820523d20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4651 [IO wait]:
net.runtime_pollWait(0x7fd402941ba0, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8200ae7d0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8200ae7d0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc8200ae770, 0xc8200a9000, 0x1000, 0x1000, 0x0, 0x7fd402936050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc8200262a0, 0xc8200a9000, 0x1000, 0x1000, 0x7fd4008a7b60, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
net/rpc.(*Client).input(0xc820101800)
	/usr/local/go/src/net/rpc/client.go:109 +0x177
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0x126

goroutine 4572 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820498d60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4578 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:613 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc820522c00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4628 [IO wait]:
net.runtime_pollWait(0x7fd402941960, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820053560, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820053560, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc820053500, 0xc8201d1000, 0x1000, 0x1000, 0x0, 0x7fd402936050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc8201065b8, 0xc8201d1000, 0x1000, 0x1000, 0xc820475900, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc82044d800, 0x7fd4008a7800, 0xc8201065b8, 0x7fd40294c4a8, 0xc8203e8780, 0xc82049bc10, 0xc820358b40)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:210 +0x6f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:213 +0xadc

goroutine 4650 [IO wait]:
net.runtime_pollWait(0x7fd402941a20, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820138f40, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820138f40, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc820138ee0, 0xc820169000, 0x1000, 0x1000, 0x0, 0x7fd402936050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc8200262a8, 0xc820169000, 0x1000, 0x1000, 0xc8204a3900, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc8202b73e0, 0x7fd4008a7800, 0xc8200262a8, 0x7fd40294c4a8, 0xc8201f3c80, 0xc8204c7c70, 0xc820101620)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:210 +0x6f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:213 +0xadc

goroutine 4609 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:556 +0x15d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc8205238c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4595 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc8205231e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4590 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc820523140)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4602 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc820523380)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4579 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:632 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc820522c20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4596 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc8202b6420, 0xece22f242, 0x18fb0b5, 0x2192c60, 0xc8204a6060, 0xc8202fad90, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x641
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:213 +0x1b0
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc820523200)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4546 [select]:
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).startStats(0xc820254820)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:214 +0xf1a
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).(github.com/cockroachdb/cockroach/kv.startStats)-fm()
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:201 +0x2e
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc82049a3b0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4565 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc820498c00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4630 [IO wait]:
net.runtime_pollWait(0x7fd402941c60, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820138a00, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820138a00, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc8201389a0, 0xc820510000, 0x1000, 0x1000, 0x0, 0x7fd402936050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc820026230, 0xc820510000, 0x1000, 0x1000, 0xc8204a5900, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc820291860, 0x7fd4008a7800, 0xc820026230, 0x7fd40294c4a8, 0xc8203e8c00, 0xc82049bc30, 0xc820358c60)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:210 +0x6f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:213 +0xadc

goroutine 4513 [select]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:213 +0x208
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc8204981e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4547 [chan receive]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8200aa190)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 4610 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:613 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc820523920)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4599 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc820523300)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4586 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:556 +0x15d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc8205230e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4552 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:613 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc8204987c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4587 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:613 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc820523100)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4560 [runnable]:
fmt.(*pp).doPrintf(0xc8203872b0, 0x181f3c0, 0xf, 0xc820544400, 0x1, 0x1)
	/tmp/workdir/go/src/fmt/print.go:1094 +0x20f
fmt.Sprintf(0x181f3c0, 0xf, 0xc820544400, 0x1, 0x1, 0x0, 0x0)
	/tmp/workdir/go/src/fmt/print.go:203 +0x7d
github.com/cockroachdb/cockroach/kv.(*DistSender).sendChunk(0xc820052770, 0x7fd402941dc8, 0xc820532ed0, 0x1a13b8607, 0x49, 0x0, 0x0, 0x0, 0x0, 0xc8203cd378, ...)
	/go/src/github.com/cockroachdb/cockroach/kv/dist_sender.go:638 +0x19a9
github.com/cockroachdb/cockroach/kv.(*DistSender).Send(0xc820052770, 0x7fd402941dc8, 0xc820532ed0, 0x1a13b8607, 0x49, 0x0, 0x0, 0x0, 0x0, 0xc8203cd378, ...)
	/go/src/github.com/cockroachdb/cockroach/kv/dist_sender.go:502 +0x30d
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).Send(0xc820254820, 0x7fd402941dc8, 0xc820532ed0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc8203cd378, ...)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:397 +0x1213
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc820498a00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4564 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc820498be0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4600 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc820523320)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4620 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:632 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb490, 0xc820523d60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4583 [IO wait]:
net.runtime_pollWait(0x7fd4029414e0, 0x72, 0x7fd402936050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8202faf40, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8202faf40, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc8202faee0, 0x0, 0x7fd402941e08, 0xc8204bb1c0)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc8200260d8, 0xc820237d20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc820561830)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4561 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc820498a20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4601 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc820523340)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4562 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc820498a40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4591 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc820523160)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4593 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fad90, 0xc8205231a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4544 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1559 +0xb0c
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fa930, 0xc820535380)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4605 [chan receive]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8202a6be0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 4606 [IO wait]:
net.runtime_pollWait(0x7fd402941660, 0x72, 0x7fd402936050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8202fb2c0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8202fb2c0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc8202fb260, 0x0, 0x7fd402941e08, 0xc820349860)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc820026100, 0xc820232d20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc820561fb0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4608 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1559 +0xb0c
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202fb110, 0xc820512330)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4548 [IO wait]:
net.runtime_pollWait(0x7fd402941420, 0x72, 0x7fd402936050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8200532c0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8200532c0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc820053260, 0x0, 0x7fd402941e08, 0xc8200d7d20)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc820106538, 0xc820033d20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200523f0, 0xc8204a7a70)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4625 [IO wait]:
net.runtime_pollWait(0x7fd4029417e0, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8201f9950, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8201f9950, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc8201f98f0, 0xc820521000, 0x1000, 0x1000, 0x0, 0x7fd402936050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc8200261f8, 0xc820521000, 0x1000, 0x1000, 0x7fd4008a7b60, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
net/rpc.(*Client).input(0xc82024be60)
	/usr/local/go/src/net/rpc/client.go:109 +0x177
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0x126

goroutine 4629 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc820291860, 0x7fd40294c4a8, 0xc8203e8c00, 0xc820358c60)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:300 +0xb5
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc820291860, 0x7fd40294c4a8, 0xc8203e8c00, 0xc820358c60)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:207 +0x4d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:208 +0xa6c

goroutine 4631 [IO wait]:
net.runtime_pollWait(0x7fd4029415a0, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820138990, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820138990, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc820138930, 0xc8204c0000, 0x1000, 0x1000, 0x0, 0x7fd402936050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc820026228, 0xc8204c0000, 0x1000, 0x1000, 0x7fd4008a7b60, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
net/rpc.(*Client).input(0xc820358d20)
	/usr/local/go/src/net/rpc/client.go:109 +0x177
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0x126

goroutine 4661 [select]:
github.com/cockroachdb/cockroach/storage.(*idAllocator).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/id_alloc.go:129 +0xbf2
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820052850, 0xc82047aa00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 4973 [IO wait]:
net.runtime_pollWait(0x7fd4029411e0, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8205ce220, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8205ce220, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc8205ce1c0, 0xc8205d5000, 0x1000, 0x1000, 0x0, 0x7fd402936050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc820106f20, 0xc8205d5000, 0x1000, 0x1000, 0xc820473900, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc8202ff980, 0x7fd4008a7800, 0xc820106f20, 0x7fd40294c4a8, 0xc82059e600, 0xc8205b92e0, 0xc82058db00)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:210 +0x6f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:213 +0xadc

goroutine 4849 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc8202ff020, 0x7fd40294c4a8, 0xc8203d4300, 0xc8203ceba0)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:300 +0xb5
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc8202ff020, 0x7fd40294c4a8, 0xc8203d4300, 0xc8203ceba0)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:207 +0x4d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:208 +0xa6c

goroutine 4850 [IO wait]:
net.runtime_pollWait(0x7fd402941360, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8203c93a0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8203c93a0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc8203c9340, 0xc82039b000, 0x1000, 0x1000, 0x0, 0x7fd402936050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc8200269e0, 0xc82039b000, 0x1000, 0x1000, 0xc8200c3900, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc8202ff020, 0x7fd4008a7800, 0xc8200269e0, 0x7fd40294c4a8, 0xc8203d4300, 0xc820392320, 0xc8203ceba0)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:210 +0x6f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:213 +0xadc

goroutine 4972 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc8202ff980, 0x7fd40294c4a8, 0xc82059e600, 0xc82058db00)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:300 +0xb5
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc8202ff980, 0x7fd40294c4a8, 0xc82059e600, 0xc82058db00)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:207 +0x4d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:208 +0xa6c

goroutine 4903 [IO wait]:
net.runtime_pollWait(0x7fd4029412a0, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8205ce1b0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8205ce1b0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc8205ce150, 0xc82039f000, 0x1000, 0x1000, 0x0, 0x7fd402936050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc820106f18, 0xc82039f000, 0x1000, 0x1000, 0x7fd4008a7b60, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
net/rpc.(*Client).input(0xc8203cfc80)
	/usr/local/go/src/net/rpc/client.go:109 +0x177
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0x126

goroutine 6452 [semacquire]:
sync.runtime_Syncsemacquire(0xc8204701d0)
	/tmp/workdir/go/src/runtime/sema.go:237 +0x201
sync.(*Cond).Wait(0xc8204701c0)
	/tmp/workdir/go/src/sync/cond.go:62 +0xb3
github.com/cockroachdb/cockroach/util/stop.(*Stopper).Quiesce(0xc820052850)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:266 +0x2a3
github.com/cockroachdb/cockroach/storage_test.(*multiTestContext).Stop.func1(0xc82000c780, 0xc820574cc0)
	/go/src/github.com/cockroachdb/cockroach/storage/client_test.go:258 +0x379
created by github.com/cockroachdb/cockroach/storage_test.(*multiTestContext).Stop
	/go/src/github.com/cockroachdb/cockroach/storage/client_test.go:270 +0x8c
FAIL	github.com/cockroachdb/cockroach/storage	37.698s
=== RUN   TestBatchBasics
I0109 12:22:05.947824 982 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchBasics (0.00s)
=== RUN   TestBatchGet
I0109 12:22:05.951068 982 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchGet (0.00s)
=== RUN   TestBatchMerge
I0109 12:22:05.956624 982 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchMerge (0.00s)
=== RUN   TestBatchProto

```
Please assign, take a look and update the issue accordingly.

--- a/storage/queue.go
+++ b/storage/queue.go
@@ -367,7 +367,6 @@ func (bq *baseQueue) processOne(clock *hlc.Clock) {
 // bq.Lock should not be held while calling this method.
 func (bq *baseQueue) processReplica(repl *Replica, clock *hlc.Clock) {
 	start := time.Now()
-	now := clock.Now()
 
 	// Load the system config.
 	cfg := bq.gossip.GetSystemConfig()
@@ -389,7 +388,7 @@ func (bq *baseQueue) processReplica(repl *Replica, clock *hlc.Clock) {
 	// and renew or acquire if necessary.
 	if bq.impl.needsLeaderLease() {
 		// Create a "fake" get request in order to invoke redirectOnOrAcquireLease.
-		if err := repl.redirectOnOrAcquireLeaderLease(nil /* Trace */, now); err != nil {
+		if err := repl.redirectOnOrAcquireLeaderLease(nil /* Trace */); err != nil {
 			bq.eventLog.Infof(log.V(3), "%s: could not acquire leader lease; skipping", repl)
 			return
 		}
@@ -397,7 +396,7 @@ func (bq *baseQueue) processReplica(repl *Replica, clock *hlc.Clock) {
 
 	bq.eventLog.Infof(log.V(3), "%s: processing", repl)
 
-	if err := bq.impl.process(now, repl, cfg); err != nil {
+	if err := bq.impl.process(clock.Now(), repl, cfg); err != nil {
 		bq.eventLog.Errorf("%s: error: %v", repl, err)
 	} else {
 		bq.eventLog.Infof(log.V(2), "%s: done: %0.2fms", repl,

--- a/storage/replica.go
+++ b/storage/replica.go
@@ -462,23 +462,11 @@ func (r *Replica) requestLeaderLease(timestamp roachpb.Timestamp) error {
 //  will fail as well. If it succeeds, as is likely, then the write
 //  will not incur latency waiting for the command to complete.
 //  Reads, however, must wait.
-func (r *Replica) redirectOnOrAcquireLeaderLease(trace *tracer.Trace, timestamp roachpb.Timestamp) error {
+func (r *Replica) redirectOnOrAcquireLeaderLease(trace *tracer.Trace) error {
 	r.llMu.Lock()
 	defer r.llMu.Unlock()
 
-	// This is subtle: suppose the following happens:
-	// - Node A acquires the leader lease
-	// - Node B attempts to service a command, and redirects to Node A
-	// - Node A dies
-	// - Node B retries, redirecting to Node A indefinitely, because Node B's
-	// leader lease cache is never invalidated, since it will always satisfy
-	// the command's timestamp, which does not advance.
-	//
-	// To deal with this, we do not consider a leader lease in effect unless it
-	// covers the greater of the command's timestamp and the current node's
-	// clock's timestamp. This is kind of a hack, but we don't have a better way
-	// of invalidating the leader lease cache.
-	timestamp.Forward(r.store.Clock().Now())
+	timestamp := r.store.Clock().Now()
 
 	if lease := r.getLease(); lease.Covers(timestamp) {
 		if lease.OwnedBy(r.store.StoreID()) {
@@ -815,7 +803,7 @@ func (r *Replica) addAdminCmd(ctx context.Context, ba roachpb.BatchRequest) (*ro
 	}
 
 	// Admin commands always require the leader lease.
-	if err := r.redirectOnOrAcquireLeaderLease(tracer.FromCtx(ctx), ba.Timestamp); err != nil {
+	if err := r.redirectOnOrAcquireLeaderLease(tracer.FromCtx(ctx)); err != nil {
 		return nil, err
 	}
 
@@ -848,7 +836,6 @@ func (r *Replica) addAdminCmd(ctx context.Context, ba roachpb.BatchRequest) (*ro
 // overlapping writes currently processing through Raft ahead of us to
 // clear via the read queue.
 func (r *Replica) addReadOnlyCmd(ctx context.Context, ba roachpb.BatchRequest) (*roachpb.BatchResponse, error) {
-	header := ba.Header
 	trace := tracer.FromCtx(ctx)
 
 	// Add the read to the command queue to gate subsequent
@@ -860,7 +847,7 @@ func (r *Replica) addReadOnlyCmd(ctx context.Context, ba roachpb.BatchRequest) (
 	// If there are command keys (there might not be if reads are
 	// inconsistent), the read requires the leader lease.
 	if len(cmdKeys) > 0 {
-		if err := r.redirectOnOrAcquireLeaderLease(trace, header.Timestamp); err != nil {
+		if err := r.redirectOnOrAcquireLeaderLease(trace); err != nil {
 			r.endCmds(cmdKeys, ba, err)
 			return nil, err
 		}
@@ -923,7 +910,7 @@ func (r *Replica) addWriteCmd(ctx context.Context, ba roachpb.BatchRequest, wg *
 	qDone()
 
 	// This replica must have leader lease to process a write.
-	if err := r.redirectOnOrAcquireLeaderLease(trace, ba.Timestamp); err != nil {
+	if err := r.redirectOnOrAcquireLeaderLease(trace); err != nil {
 		r.endCmds(cmdKeys, ba, err)
 		return nil, err
 	}
@@ -1558,9 +1545,8 @@ func (r *Replica) getLeaseForGossip(ctx context.Context) (bool, error) {
 	var hasLease bool
 	var err error
 	if !r.store.Stopper().RunTask(func() {
-		timestamp := r.store.Clock().Now()
 		// Check for or obtain the lease, if none active.
-		err = r.redirectOnOrAcquireLeaderLease(tracer.FromCtx(ctx), timestamp)
+		err = r.redirectOnOrAcquireLeaderLease(tracer.FromCtx(ctx))
 		hasLease = err == nil
 		if err != nil {
 			switch e := err.(type) {

--- a/storage/replica.go
+++ b/storage/replica.go
@@ -466,6 +466,20 @@ func (r *Replica) redirectOnOrAcquireLeaderLease(trace *tracer.Trace, timestamp 
 	r.llMu.Lock()
 	defer r.llMu.Unlock()
 
+	// This is subtle: suppose the following happens:
+	// - Node A acquires the leader lease
+	// - Node B attempts to service a command, and redirects to Node A
+	// - Node A dies
+	// - Node B retries, redirecting to Node A indefinitely, because Node B's
+	// leader lease cache is never invalidated, since it will always satisfy
+	// the command's timestamp, which does not advance.
+	//
+	// To deal with this, we do not consider a leader lease in effect unless it
+	// covers the greater of the command's timestamp and the current node's
+	// clock's timestamp. This is kind of a hack, but we don't have a better way
+	// of invalidating the leader lease cache.
+	timestamp.Forward(r.store.Clock().Now())
+
 	if lease := r.getLease(); lease.Covers(timestamp) {
 		if lease.OwnedBy(r.store.StoreID()) {
 			// Happy path: We have an active lease, nothing to do.

--- a/storage/replica_command.go
+++ b/storage/replica_command.go
@@ -60,6 +60,12 @@ func (r *Replica) executeCmd(batch engine.Engine, ms *engine.MVCCStats, h roachp
 		}
 	}
 
+	// Update the node clock with the serviced request. This maintains a
+	// high water mark for all ops serviced, so that received ops
+	// without a timestamp specified are guaranteed one higher than any
+	// op already executed for overlapping keys.
+	r.store.Clock().Update(ts)
+
 	var reply roachpb.Response
 	var intents []roachpb.Intent
 	var err error
@@ -147,12 +153,6 @@ func (r *Replica) executeCmd(batch engine.Engine, ms *engine.MVCCStats, h roachp
 	if log.V(2) {
 		log.Infof("executed %s command %+v: %+v, err=%s", args.Method(), args, reply, err)
 	}
-
-	// Update the node clock with the serviced request. This maintains a
-	// high water mark for all ops serviced, so that received ops
-	// without a timestamp specified are guaranteed one higher than any
-	// op already executed for overlapping keys.
-	r.store.Clock().Update(ts)
 
 	// Propagate the request timestamp (which may have changed).
 	// TODO(tschottdorf): really? Think this should be done by executeBatch.

--- a/storage/replica_test.go
+++ b/storage/replica_test.go
@@ -472,7 +472,7 @@ func TestRangeLeaderLease(t *testing.T) {
 		t.Errorf("expected another replica to have leader lease")
 	}
 
-	err := tc.rng.redirectOnOrAcquireLeaderLease(nil, tc.clock.Now())
+	err := tc.rng.redirectOnOrAcquireLeaderLease(nil)
 	if lErr, ok := err.(*roachpb.NotLeaderError); !ok || lErr == nil {
 		t.Fatalf("wanted NotLeaderError, got %s", err)
 	}
@@ -495,7 +495,7 @@ func TestRangeLeaderLease(t *testing.T) {
 		}
 	}
 
-	if _, ok := rng.redirectOnOrAcquireLeaderLease(nil, tc.clock.Now()).(*roachpb.NotLeaderError); !ok {
+	if _, ok := rng.redirectOnOrAcquireLeaderLease(nil).(*roachpb.NotLeaderError); !ok {
 		t.Fatalf("expected %T, got %s", &roachpb.NotLeaderError{}, err)
 	}
 }

--- a/storage/replica_test.go
+++ b/storage/replica_test.go
@@ -1026,7 +1026,7 @@ func TestAcquireLeaderLease(t *testing.T) {
 		// matic lease for us at the beginning, we'll basically create a lease from
 		// then on.
 		expStart := tc.rng.getLease().Start
-		tc.manualClock.Set(int64(DefaultLeaderLeaseDuration + 1000))
+		tc.manualClock.Increment(int64(DefaultLeaderLeaseDuration + 1000))
 
 		ts := tc.clock.Now()
 		if _, err := client.SendWrappedWith(tc.Sender(), tc.rng.context(), roachpb.Header{Timestamp: ts}, test); err != nil {
@@ -1038,8 +1038,8 @@ func TestAcquireLeaderLease(t *testing.T) {
 		lease := tc.rng.getLease()
 		// The lease may start earlier than our request timestamp, but the
 		// expiration will still be measured relative to it.
-		expExpiration := ts.Add(int64(DefaultLeaderLeaseDuration), 0)
-		if !lease.Start.Equal(expStart) || !lease.Expiration.Equal(expExpiration) {
+		expExpiration := ts.Next().Add(int64(DefaultLeaderLeaseDuration), 0)
+		if !(lease.Start.Equal(expStart) && lease.Expiration.Equal(expExpiration)) {
 			t.Errorf("%d: unexpected lease timing %s, %s; expected %s, %s", i,
 				lease.Start, lease.Expiration, expStart, expExpiration)
 		}

